src/service.js

```diff
@@ -771,8 +771,12 @@ class Service {
 	 * @param {String|Number?} version
 	 */
 	static getVersionedFullName(name, version) {
-		if (version != null)
-			return (typeof(version) == "number" ? "v" + version : version) + "." + name;
+		if (version != null) {
+			if (typeof(version) === 'number' || !isNaN(Number(version))) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/moleculerjs/moleculer/pullRequest?prid=8141838)
```suggestion
			if (typeof(version) === "number" || !isNaN(Number(version))) {
```